mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraUserMailboxRightsDAO.java

```diff
@@ -98,10 +100,11 @@ private PreparedStatement prepareSelectAllForUser(Session session) {
     }
 
     public CompletableFuture<Void> update(CassandraId cassandraId, PositiveUserACLChanged positiveUserAclChanged) {
+        ACLDiff aclDiff = ACLDiff.from(positiveUserAclChanged.getOldACL(), positiveUserAclChanged.getNewACL());
```

Comment:
Maybe I would rename "from" to "diff" because afterall it's what you compute a diff